### PR TITLE
Add ability for a user to change from voter to observer

### PR DESCRIPTION
**Pull request by newportt**
Screenshot from a voter's perspective:
screen shot 2014-03-16 at 12 18 52 pm

Closes #7

### CODE REVIEW

**Comment by zacharytamas** [comment]
Where are the tests? :)

:+1:

File: censeo/templates/censeo/snippets/users.html [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+    <li class="user">
+      {% get_full_name_or_username observer %}
+      <span class="pull-right label label-info">
+        <i class="glyphicon glyphicon-eye-open"></i>
+      </span>
+    </li>
   {% endfor %}
 </ul>
+{% if user in voters %}
+  <a href="{% url 'become_observer' meeting.id %}" class="btn btn-info become-observer">
```
Comment (cjflory) [inline censeo/templates/censeo/snippets/users.html]: What do you think about adding `btn-xs` to this button and the [reset votes button](https://github.com/cjflory/censeo/blob/develop/censeo/templates/censeo/snippets/voting.html#L24)?

**Comment by cjflory** [comment]
:+1:  Looks awesome!

File: censeo/templates/censeo/snippets/users.html (hunk 1 above)
Comment (newportt) [inline censeo/templates/censeo/snippets/users.html]: Smaller buttons definitely look better, just updated it

**Pull request merged**